Frameworks/UIKit/StarboardXaml/ApplicationMain.mm

```diff
@@ -94,6 +96,10 @@ int ApplicationMainStart(const char* principalName,
     WOCDisplayMode* displayMode = [UIApplication displayMode];
     [displayMode _setWindowSize:CGSizeMake(windowWidth, windowHeight)];
 
+    if (activationType == ActivationTypeLibrary) {
+        [displayMode setDisplayPreset:WOCDisplayPresetNative];
```

Comment:
That would require me to understand why it's necessary...

---
In reply to: [88144728](https://github.com/Microsoft/WinObjC/pull/1393#discussion_r88144728) [](ancestors = 88144728)